core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -206,11 +206,21 @@ func (dq *decryptionQueue) SetResult(ciphertextId decryptionPlugin.CiphertextId,
 
 	req, ok := dq.pendingRequests[string(ciphertextId)]
 	if ok {
-		dq.lggr.Debugf("responding with result for pending decryption request ciphertextId %s", string(ciphertextId))
-		req.chPlaintext <- plaintext
+		if err != nil {
+			dq.lggr.Debugf("decryption error for ciphertextId %s", string(ciphertextId))
+		} else {
+			dq.lggr.Debugf("responding with result for pending decryption request ciphertextId %s", string(ciphertextId))
+			req.chPlaintext <- plaintext
+		}
 		close(req.chPlaintext)
 		delete(dq.pendingRequests, string(ciphertextId))
 	} else {
+		if err != nil {
+			// This is currently possible only for ErrAggregation, encountered during Report() phase.
+			dq.lggr.Debugf("received decryption error for ciphertextId %s which doesn't exist locally", string(ciphertextId))
```

Comment:
Same as above, why suppress the error message?

Reply:
See above.